osbs/build/build_response.py

```diff
@@ -119,6 +119,19 @@ def get_logs(self, decode_logs=True):
         else:
             return logs
 
+    def get_error_message(self):
+        try:
+            str_metadata = graceful_chain_get(self.get_annotations_or_labels(), "plugins-metadata")
+            metadata_dict = json.loads(str_metadata)
+            error = metadata_dict['errors'].items()[0]
+            if error[1]:
+                # Plugin has non-empty error description
+                return "Error in plugin %s: %s" % (error[0], error[1])
+            else:
+                return "Error in plugin %s" % (error[0])
+        except:
+            return ""
```

Comment:
Should we return `None` when there is no error?

Would be good to have a docstring for the method too.